CHANGELOG.md

```diff
@@ -30,6 +21,20 @@ modified Semantic Versioning scheme. See the "Versioning scheme" section of the
 
 ### Security
 
+## [1.7.5] - 2022-08-11
+### Changed
+- RIGA-282: Update drupal/core 9.3.19 => 9.4.5.
+- RIGA-282: Update core patch for issue 1356276.
+- RIGA-282: Update line numbers of htaccess patch.
+- RIGA-297: Upgrade drupal/acsf_duplication 2.68.0 => 2.72.0.
+- RIGA-297: Upgrade drupal/acsf_theme 2.68.0 => 2.72.0.
+- RIGA-297: Upgrade drupal/acsf_variables 2.68.0 => 2.72.0.
+- RIGA-297: Upgrade drupal/memcache 2.3.0 => 2.4.0.
+- RIGA-297: Upgrade drupal/search_api 1.23.0 => 1.25.0.
+- RIGA-297: Upgrade drupal/search_api_solr 4.2.7 => 4.2.8.
+- RIGA-297: Upgrade drupal/token 1.10.0 => 1.11.0.
+- RIGA-297: Upgrade drupal/jsonapi_extras 3.20.0 => 3.21.0.
+
```

Comment:
@swiftwinning almost! Just need a line noting the update to the ecms_profile, similar to last release:
```RIGA-6: Updated ecms_profile to 0.9.12.```